lib/matplotlib/axes/_axes.py

```diff
@@ -3284,6 +3284,18 @@ def errorbar(self, x, y, yerr=None, xerr=None,
         if len(x) != len(y):
             raise ValueError("'x' and 'y' must have the same size")
 
+        def check_if_negative(array):
```

Comment:
Naming suggestion:

```suggestion
        def has_negative_values(array):
```

Reply:
That's indeed better. Thanks @timhoffm

Reply:
@timhoffm are you OK, If I "Resolve conversation" under the things that are addressed?